config/initializers/friendly_id.rb

```diff
@@ -1,6 +1,12 @@
 FriendlyId.defaults do |config|
-  config.base = :name
-  config.use :slugged, :finders, FriendlyId::SequentialSlugs
+  module CustomNormalisation
+    def normalize_friendly_id(input)
+      # Use the babosa gem for better string parameterisation
+      super input.to_s.to_slug.truncate(150).normalize.to_s
+    end
+  end
 
+  config.base = :name
+  config.use :sequentially_slugged, :finders, CustomNormalisation
```

Comment:
Are we using `:finders`? In the work to upgrade to 4.2.2 I changed all the calls to be `Model.friendly.find`.